examples/footer/index.html

```diff
@@ -8,59 +8,85 @@
   </head>
 
   <body>
-    <footer class="footer">
-      <div class="container-fluid">
-        <p class="copyright">
-          © Copyright 2020. St. Jude Children's Research Hospital, a not-for-profit, section 501(c)(3).
-          <a href="https://www.stjude.org" target="_blank" rel="noopener noreferrer">www.stjude.org</a> |
-          <a href="https://stjude.cloud/contact" target="_blank" rel="noopener noreferrer">Contact Us</a> |
-          <a href="https://stjude.cloud/privacy-policy">Privacy Policy</a> |
-          <a href="https://stjude.cloud/terms-of-use">Terms and Conditions</a>
-        </p>
-        <ul class="nav justify-content-center">
-          <li>
-            <a target="_blank" rel="noopener noreferrer" href="https://www.stjude.org/legal/u-s-privacy-policy-statement.html" title="U.S. Privacy Policy Statement">U.S. Privacy Policy Statement</a>
-          </li>
-          <li>
-            <a target="_blank" rel="noopener noreferrer" href="https://www.stjude.org/legal.html" title="Disclaimer / Registrations / Copyright Statement">Disclaimer / Registrations / Copyright Statement</a>
-          </li>
-          <li>
-            <a target="_blank" rel="noopener noreferrer" href="https://www.stjude.org/legal/linking-policy.html" title="Linking Policy">Linking Policy</a>
-          </li>
-          <li>
-            <a target="_blank" rel="noopener noreferrer" href="https://www.stjude.org/legal/notice-of-privacy-practices.html" title="Notice of Privacy Practices (HIPAA)">Notice of Privacy Practices (HIPAA)</a>
-          </li>
-          <li>
-            <a target="_blank" rel="noopener noreferrer" href="https://www.stjude.org/legal/notice-of-non-discrimination.html" title="Notice of Non-Discrimination">Notice of Non-Discrimination</a>
-          </li>
-        </ul>
-        <ul class="social-media">
-          <li>
-            <a target="_blank" rel="noopener noreferrer" href="https://www.facebook.com/stjuderesearch" class="fab fa-facebook-f">
-              <span class="sr-only">Facebook</span>
+    <footer>
+      <div class="container">
+        <div class="row">
+          <div class="col-6 col-lg-3 col-md-4 offset-xl-0 offset-lg-5 offset-md-4 offset-md-3 offset-3 footer-logo">
+            <a class="navbar-brand" href="https://stjude.cloud">
```

Comment:
If it doesn't make it too complex, I would pull out the styles associated with the St. Jude Child logo and St. Jude Cloud title (`.navbar-brand` and `.sjc-title`) from `scss/_sjc-navbar.scss` [here](https://github.com/stjudecloud/theme/blob/f65a86f6d16e9f4b9b0fefae822a7eeaa1e0cb2c/scss/_sjc-navbar.scss#L11-L13) and [here](https://github.com/stjudecloud/theme/blob/f65a86f6d16e9f4b9b0fefae822a7eeaa1e0cb2c/scss/_sjc-navbar.scss#L77-L89) (leave [this](https://github.com/stjudecloud/theme/blob/f65a86f6d16e9f4b9b0fefae822a7eeaa1e0cb2c/scss/_sjc-navbar.scss#L228-L232) in the navbar as it is contextually important to the navbar). I would create a separate component out of it. Creating a new component for this de-duplicates some of the styling you've defined for the footer and adds some reusability.

We should also avoid word wrapping the brand if possible:
<img width="393" alt="Screen Shot 2020-03-27 at 5 27 25 PM" src="https://user-images.githubusercontent.com/2433649/77805652-7290ac80-7050-11ea-8034-57035a456e0e.png">